common/shareddb.cpp

```
@@ -358,69 +389,46 @@ bool SharedDatabase::UpdateSharedBankSlot(uint32 char_id, const EQ::ItemInstance
-	);
+	return QueryDatabase(fmt::format(
+		"DELETE FROM inventory WHERE character_id = {} AND "
+		"(slot_id = {} OR slot_id BETWEEN {} AND {})",
+		char_id,
+		slot_id,
+		base_slot_id,
+		base_slot_id + (EQ::invbag::SLOT_COUNT - 1)
+	)).Success();
 }
 
 bool SharedDatabase::DeleteSharedBankSlot(uint32 char_id, int16 slot_id)
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)